components/ui/Tag.jsx

```diff
@@ -0,0 +1,11 @@
+const Tag = ({ content }) => {
+  return (
+    <div>
+      <p className="font-neuf text-lg text-primaryGray-400  -tracking-wide  bg-primaryGray-200  rounded-base  px-2.5  py-sm w-fit">
+        {content}
+      </p>
+    </div>
```

Comment:
You introduced an unnecessary tag. `<div>` doesn't have any semantic meaning to your code and your code should still work without the outer div.